module2/homework/nwdNww/nwdNww.hpp

```diff
@@ -1,11 +1,24 @@
 #pragma once
 
 int NWD(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+    unsigned int t;
+    while (rhs)
+    {
+        t = rhs;
+        rhs = lhs % rhs;
+        lhs = t;
+    }
+    return (lhs>0) ? lhs : lhs * (-1) ;
 }
-
 int NWW(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
-}
+if (lhs==0 || rhs==0) {return 0;}
+long long mul=lhs*rhs;
+   unsigned int t;
```

Comment:
```suggestion
    if (lhs==0 || rhs==0) {
        return 0;
    }
    long long mul=lhs*rhs;
    unsigned int t;
```
Indendation. Please avoid one-liner if-returns.